src/App.jsx

```diff
@@ -3,26 +3,52 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
```

Comment:
as the task description says:

> Create a separate file api.js to put all the API call there.

Let's create the file `api.js` and write all api functions there. And then import them from this file